src/editor/editorwithui.jsdoc

```diff
@@ -38,18 +40,37 @@
  * come with a single "main" HTML element (its editable element and toolbar are separate).
  *
  * This property can be understood as a shorthand for retrieving the element that a specific editor integration
- * considers to be its main DOM element. There are always other ways to access these elements, too
- * (e.g. via {@link #ui `editor.ui`}).
+ * considers to be its main DOM element.
  *
+ * @deprecated v12.0.0 The {@link module:core/editor/editorui~EditorUI#element `EditorUI element`} should be used instead.
  * @readonly
  * @member {HTMLElement|null} #element
  */
 
 /**
+ * **Deprecated** since `v12.0.0`. The {@link module:core/editor/editorui~EditorUI#event:ready ui ready event} should be listened to instead.
+ *
  * Fired when the editor UI is ready.
  *
  * Fired after {@link module:core/editor/editor~Editor#event:pluginsReady} and before
  * {@link module:core/editor/editor~Editor#event:dataReady}.
  *
+ * @deprecated v12.0.0 The {@link module:core/editor/editorui~EditorUI#event:ready ui ready event} should be listened to instead.
  * @event uiReady
  */
+
+/**
+ * This error is thrown when a component tries to access deprecated
+ * {@link module:core/editor/editorwithui~EditorWithUI#element editor element}. Instead the
+ * {@link module:core/editor/editorui~EditorUI#element `EditorUI element`} should be used.
+ *
+ * @error deprecated-editor-element
+ */
```

Comment:
Is it an error? Isn't it just a warning in the console?

Reply:
Without `@error` annotation there won't be an error docs generated for it, where it can be explained in more depth. Using format like `error-name: Info` with [`log.warn`](https://github.com/ckeditor/ckeditor5-utils/blob/e941872275c5c6d539cb8fb86b492123c8a5e2ed/src/log.js#L71) [automatically attaches link to error docs](https://github.com/ckeditor/ckeditor5-utils/blob/e941872275c5c6d539cb8fb86b492123c8a5e2ed/src/ckeditorerror.js#L78) so its good when the link points to existing error.

And from what I see in the code, `log.warn` is usually documented like that.

Reply:
That's fine, then.